### PR TITLE
docs: add issues section

### PR DESCRIPTION
**Pull request by HenriqueZup**
Signed-off-by: Henrique Rodrigues <henrique.rodrigues@zup.com.br>

**- What I did**
I added the issues section
**- How to verify it**
Access the CONTRIBUTING.md file of this branch, check section 2.3
**- Description for the changelog**
Adding an issue section as a prerequisite before contributing with a PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,6 +69,51 @@ For this method, your name and e-mail must be the same registered on your GitHub
...
+
+**With the title:**
+
+**[Project][Escope]: Title Description**  
+
+**Project:** Name of the project or repository you want to contribute to.  
+
+**Escope:** Add what your issue refers to:
+
+- **[Burg report](https://github.com/ZupIT/horusec/issues/new?assignees=&labels=bug&template=bug_report.md):** Create a report to help us improve
```
Comment (iancardosozup) [inline CONTRIBUTING.md:99]: typo

**Review by iancardosozup — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (iancardosozup) [reply CONTRIBUTING.md:99]: there is also a typo in `Burg report`

**Review by nathanmartinszup — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -66,6 +69,51 @@ For this method, your name and e-mail must be the same registered on your GitHub
...
+- **Issues that need extra attention:** `is:open is:issue label:"help wanted" `
+- **Issues related to a bug:** `is:issue is:open label:kind/bug `
+
+#### **Open an issue for any new problem**
+
+Writing a good issue will help our team better analyze and manage your contributions, therefore, follow the standards and best practices below:
+
+**With the title:**
+
+**[Project][Scope]: Title Description**
```
Comment (matheusalcantarazup) [inline CONTRIBUTING.md:93]: I think we can change to "Project:Scope - Tittle description" so we can follow the same pattern [here](https://github.com/ZupIT/horusec/blob/bbbb2d1f40ed0145b865c04e359179de5219e0a7/CONTRIBUTING.md#pkgtype---description)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,6 +69,51 @@ For this method, your name and e-mail must be the same registered on your GitHub
...
+
+- **[Bug report](https://github.com/ZupIT/horusec/issues/new?assignees=&labels=bug&template=bug_report.md):** Create a report to help us improve
+
+- **[Feature request](https://github.com/ZupIT/horusec/issues/new?assignees=&labels=feature&template=feature_request.md):** Suggest a new feature for a project
+
+- **[Improvement](https://github.com/ZupIT/horusec/issues/new?assignees=&labels=improvement&template=improvement.md):** Suggest a improvement for a project
+
+- **[Support request](https://github.com/ZupIT/horusec/issues/new?assignees=&labels=support&template=support_request.md):** Support request or question relating to Horusec
+
+**Example: [Horusec-cli][Improvement]: Suggestion for CLI installation experience**
```
Comment (matheusalcantarazup) [inline CONTRIBUTING.md:107]: So this example would be changed to `Horusec-cli:Improvement: Suggestion for CLI installation experience`

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]
Btw, do we really need this scope and project? Isn't that the function of tags?

cc @nathanmartinszup @wiliansilvazup @iancardosozup

**Review by iancardosozup — APPROVED** [review]

**Comment by wiliansilvazup** [comment]
> Btw, do we really need this scope and project? Isn't that the function of tags?
> 
> cc @nathanmartinszup @wiliansilvazup @iancardosozup

Yeah, this was arranged with the technical writer team!

**Review by matheusalcantarazup — APPROVED** [review]

**Pull request merged**